install.sh

```diff
@@ -449,6 +449,11 @@ pushd .
     cmake_common_options="${cmake_common_options} -DCMAKE_BUILD_TYPE=Debug"
   fi
 
+  # cuda
+  if [[ "${build_cuda}" == false ]]; then
+    cmake_common_options="${cmake_common_options} -DTRY_CUDA=OFF"
```

Comment:
rocsparse has a flag as well. it is called use-cuda. Should we make it consistent.

Reply:
If others agree I can change it, this is just fixing my ability to build without cuda when cuda is partially present.
rocBLAS also is install.sh with --use-cuda, and -DUSE_CUDA.

Reply:
Can someone look at the jenkins ?   I still see this CI build error: mv: 'hipblas-0.37.0.434-PR-256-1-4ab1946-ubuntu-18.04_amd64.deb' and 'hipblas-0.37.0.434-PR-256-1-4ab1946-ubuntu-18.04_amd64.deb' are the same file
script returned exit code 1